docs/cortex/installation-and-configuration/deploy-cortex-on-kubernetes.md

```diff
@@ -0,0 +1,242 @@
+            - cortex-shared-fs-claim
+          volumeMounts:
+            # Must match the name defined in the volumes section
+            - name: cortex-job-pvc
+              mountPath: /tmp/cortex-jobs
+          # (...)
+      volumes:
+        - name: cortex-job-pvc
+          persistentVolumeClaim:
+            # Reference the previously defined PVC
+            claimName: cortex-shared-fs-claim
+```
+
+## Step 4: Verify the deployment and service account
+
+Run the following command to ensure the deployment is running with the correct SA:
+
+```bash
+kubectl get deployment cortex -o=jsonpath='{.spec.template.spec.serviceAccountName}'
+```
+
+Verify the PVC is bound:
+
+```bash
+kubectl get pvc cortex-shared-fs-claim
+```
+
+## Example: Deploy Cortex using AWS EFS
+
+### Prerequisites
+
+Before setting up the PV for AWS EFS, complete the following steps:
+
+1. [Create an Identity and Access Management (IAM) role](https://docs.aws.amazon.com/eks/latest/userguide/efs-csi.html) to allow the EFS CSI driver to interact with EFS.
+2. Install the EFS CSI driver on your Kubernetes cluster using one of the following methods:
+    * [EKS add-ons](https://www.eksworkshop.com/docs/fundamentals/storage/efs/efs-csi-driver) (recommended)
+    * [Official Helm Chart](https://github.com/kubernetes-sigs/aws-efs-csi-driver/releases?q=helm-chart&expanded=true)
+3. [Create an EFS filesystem](https://github.com/kubernetes-sigs/aws-efs-csi-driver/blob/master/docs/efs-create-filesystem.md) and note the associated EFS filesystem ID.
+
+### 1. Create a StorageClass for EFS
+
+!!! note "Reference example"
+    The following manifests are based on the [EFS CSI driver multiple pods example](https://github.com/kubernetes-sigs/aws-efs-csi-driver/tree/master/examples/kubernetes/multiple_pods).
+
+Create a StorageClass that references your EFS filesystem:    
+
+```yaml
+kind: StorageClass
+apiVersion: storage.k8s.io/v1
+metadata:
+  name: efs-sc
+provisioner: efs.csi.aws.com
+# https://github.com/kubernetes-sigs/aws-efs-csi-driver?tab=readme-ov-file#storage-class-parameters-for-dynamic-provisioning
+parameters:
+  provisioningMode: efs-ap # EFS access point provisioning mode
+  fileSystemId: fs-01234567 # Replace with your EFS filesystem ID
+  directoryPerms: "700" # Permissions for newly created directories
+  uid: 1001 # User ID to set file permissions
+  gid: 1001 # Group ID to set file permissions
+  ensureUniqueDirectory: "false" # Set to false to allow shared folder access between Cortex and job containers
+  subPathPattern: "${.PVC.namespace}/${.PVC.name}" # Optional subfolder structure inside the NFS filesystem
+```
+
+### 2. Create a PVC using the EFS StorageClass
+
+Kubernetes automatically creates a PV when defining a PVC with the EFS StorageClass.
+
+Define the PVC as follows:
+
+```yaml
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: cortex-shared-fs-claim
+spec:
+  storageClassName: efs-sc # References the EFS StorageClass
+  # (...)
+```
```

Comment:
This illustrates how an admin can manually define the PVC, but in Cortex Helm Chart case we can simply let the chart create the PVC by simply filling the correct values (in particular, the `storageClass` key):

https://github.com/StrangeBeeCorp/helm-charts/blob/cortex-initial-helm-chart/cortex-charts/cortex/values.yaml#L67-L79